engine/src/mccontrol.h

```diff
@@ -323,6 +323,11 @@ class MCControl : public MCObject
 	{
 		return (MCControl *)MCDLlist::remove((MCDLlist *&)list);
 	}
+    
+    virtual uint16_t getminsize() const
```

Comment:
I'd move this up to near the other MCControl introduced virtuals... There's no point in having it 'inline' as it needs to be placed as a function ptr in the vtable (so it will never be inlined) - the implementation should go in control.cpp.